packages/wrangler/templates/pages-dev-pipeline.ts

```diff
@@ -2,8 +2,9 @@
 import worker from "__ENTRY_POINT__";
 // @ts-ignore entry point will get replaced
 export * from "__ENTRY_POINT__";
+import { isRoutingRuleMatch } from "./pages-dev-util";
 
-// @ts-ignore routes and transformRuleToRegExp are injected
+// @ts-ignore routes are injected
 const routes = __ROUTES__;
 
 export default {
```

Comment:
Don't think you need the `@ts-ignore`s on L14 and L21 anymore?

Reply:
could be good to use ts-expect-error so it warns you if there's not an error there?

Reply:
I'd expect `__ROUTES__` to always be injected here? I can't think of a scenario in which we'd want to internally override that variable here 🤔